cves/CVE-2010-4704.yml

```diff
@@ -171,10 +193,12 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then please
     explain where you looked. Thus, 'answer' should always have some
     explanation.
-  answer:
-  automated:
-  contest:
-  developer:
+  answer: |
+    This vulnerability discovery is credited to Gregory Maxwell and was found using 'zzuf'
+    Zuff is a fuzzer that randomly sets a programs inputs to look for vulnerabilities
```

Comment:
I would also add that it's an all-purpose fuzzer, meaning it's not aware of media transcoding.